community/louvain.tex

```diff
@@ -7,15 +7,19 @@
 \usepackage{amsmath,amsfonts}
 \usepackage[margin=4cm]{geometry}
 
+\title{Louvain algorithm for undirected and directed graphs}
+
 \begin{document}
 
+\maketitle
+
 The algorithm attempts to find communities (highly connected sub-graphs),
 and it does this by minimising the modularity function
 \begin{equation}
-	Q = \frac{1}{2m}\sum_i\sum_j\left[ A_{ij} - \gamma \frac{k_ik_j}{2m} \right] \delta(c_i,c_j),
+	Q(c) = \frac{1}{2m}\sum_i\sum_j\left[ A_{ij} - \gamma \frac{k_ik_j}{2m} \right] \delta_{ij}(c),
 \end{equation}
 where $A_{ij}$ is the edge weight between nodes $i$ and $j$, 
-$c_i$ is the current community to which node $i$ belongs,
+$c(i)$ is the current community to which node $i$ belongs,
```

Comment:
If I were to nitpick, I would say that since c(i) does not appear in the formula anymore, I would replace this line with a description of what c itself is (`$c$ is a partition of nodes into disjoint subsets or communities`) and I would put it after `where`.